client/css/style.css

```diff
@@ -775,6 +775,10 @@ kbd {
 	width: 100%;
 }
 
+.input:disabled {
+	background-color: #e9ecef;
```

Comment:
That would be a new shade of gray (-ish) in our CSS. Could we re-use one maybe instead? `#eee` seems very close and we use it already. Or better if you prefer something else.

Reply:
`#eee` is way too light, on one of my non-IPS displays I can't tell the difference.